src/Symfony/Component/Console/Tests/Question/ConfirmationQuestionTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Tests\Question;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+class ConfirmationQuestionTest extends TestCase
+{
+    /**
+     * @dataProvider normalizerUsecases
+     */
+    public function testDefaultRegexUsecases($default, $answers, $expected, $message)
+    {
+        $sut = new ConfirmationQuestion('A question', $default);
+
+        foreach ($answers as $answer) {
+            $actual = $sut->getNormalizer()($answer);
```

Comment:
3.4 supports PHP 5.5, the result of `getNormalizer()` needs to be assigned before being called (see travis)

Reply:
ouch :) nice catch.